src/ape_plugins/_cli.py

```diff
@@ -22,38 +23,80 @@ def cli():
     """
 
 
-@cli.command(name="list", short_help="List installed plugins")
+def _format_output(plugins_list: List[str], version: Optional[str]) -> Set:
+    output = set()
+    prefix = "ape_"
+    for i in plugins_list:
+        if i.startswith(prefix):
+            start = len(prefix)
+            text = i[start:]
+        else:
+            text = i
+        if version:
+            output.add(f"{text}    ({version})")
+        else:
+            output.add(text)
+    return output
+
+
+@cli.command(name="list", short_help="Display plugins")
 @click.option(
     "-a",
     "--all",
     "display_all",
     default=False,
     is_flag=True,
-    help="Display all plugins (including Core)",
+    help="Display all plugins installed and available (including Core)",
 )
 @ape_cli_context()
 def _list(cli_ctx, display_all):
-    plugins = []
+    installed_second_class_plugins = set()
+    installed_first_class_plugins = set()
```

Comment:
A lol nit, I think it is funny that these are in order 2, 1, 3 rather than 1, 2, 3